internal/log/log.go

```diff
@@ -183,6 +189,11 @@ func (l Logger) logFail(format string, a ...interface{}) {
 	l.Crit(fmt.Sprintf("[failed]   %s", format), a...)
 }
 
+// logFail logs the failure of a multi-step/substantial/time-consuming operation.
+func (l Logger) logAborted(format string, a ...interface{}) {
+	l.Warning(fmt.Sprintf("[aborted]   %s", format), a...)
```

Comment:
Hmm, I wonder if this should be bumped down to a `Info` or even a `Debug` message. It feels like this might cause too much noise for what is almost always a relatively benign by-product of multi-fetches.